Sprint-1/exercises/random.js

```diff
@@ -7,3 +7,15 @@ const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
 // Try breaking down the expression and using documentation to explain what it means
 // It will help to think about the order in which expressions are evaluated
 // Try logging the value of num and running the program several times to build an idea of what the program is doing
+
+/*
+    (maximum - minimum + 1) = 100
```

Comment:
What is the purpose behind the multiplication?

Reply:
The purpose is to scale the random value